bootstrap

```diff
@@ -3,23 +3,10 @@
 # This script sets up a brand-new FreeBSD host as the proper piece of Buildbot
 # infrastructure.  See README.rst
 #
-# Usage: ./bootstrap HOST
+# Usage: ./bootstrap
 
 set -e
 
-HOSTNAME="${1}"
-if [ -z "${HOSTNAME}" ]; then
-    echo "USAGE: ./bootstrap hostname"
-    exit 1
-fi
+env ASSUME_ALWAYS_YES=YES pkg install ansible git python2
 
-SSH_ARGS="$SSH_ARGS -o PreferredAuthentications=password,keyboard-interactive"
-# don't prompt to add the SSH key
-SSH_ARGS="$SSH_ARGS -o StrictHostKeyChecking=no"
-# use a control connection to send multiple SSH commands through a single connection
-SSH_ARGS="$SSH_ARGS -o ControlPersist=10m"
-SSH_ARGS="$SSH_ARGS -o ControlMaster=auto"
-
-export ANSIBLE_SSH_ARGS="$SSH_ARGS"
-
-ansible-playbook --ask-vault-pass -e target_host=${HOSTNAME} bootstrap.yml
+ansible-playbook --ask-vault-pass local.yml
```

Comment:
what about `bootstrap.yml`?

Reply:
Haven't decided what to do with it. It requires remote access with SSH using a password. That works fine if it's a strong password but not so great when it's a weak password (such as with the recent vm1 re-install). My assumption was that it's easier to do an install, give a weak root password for `su`, and restrict user logins to using keys.

Reply:
Could we just require that the user's ssh key is set up on the remote host (e.g., with ssh-copy-id)?  Then passwords are irrelevant..